app/src/main/java/com/sdp/movemeet/ui/workout/ListViewAdapter.java

```diff
@@ -0,0 +1,106 @@
+package com.sdp.movemeet.ui.workout;
+
```

Comment:
What is the goal of a list view adapter ?

Reply:
It basically takes care of displaying the list of available workouts.